frontend/src/components/home-page/content-tree.scss

```diff
@@ -1,5 +1,6 @@
 .content-tree {
   width: 40%;
+  max-width: 400px;
```

Comment:
This should be a bit wider probably. It makes the questions pretty squished.

Reply:
i thought i removed this smh